bibblio.py

```diff
@@ -312,7 +463,12 @@ def get_full_text(self, work):
             .join(LicensePoolDeliveryMechanism.delivery_mechanism)\
             .filter(
                 LicensePool.work_id==work.id,
-                DeliveryMechanism.drm_scheme==DeliveryMechanism.NO_DRM)\
+                DeliveryMechanism.drm_scheme==DeliveryMechanism.NO_DRM,
+                or_(
+                    Representation.status_code==200,
+                    Representation.status_code==None
+                )
+            )\
```

Comment:
This is the kind of thing you might be able to bypass with `restrict_to_ready_deliverable_works`.